django/template/defaultfilters.py

```diff
@@ -134,10 +134,7 @@ def floatformat(text, arg=-1):
     if not m and p < 0:
         return mark_safe(formats.number_format('%d' % (int(d)), 0))
 
-    if p == 0:
-        exp = Decimal(1)
-    else:
-        exp = Decimal('1.0') / (Decimal(10) ** abs(p))
+    exp = Decimal(10).scaleb(-abs(p))
```

Comment:
It should be `Decimal(1)`, not `Decimal(10)`, not sure why tests pass.

Reply:
@sir-sigurd You're right 🤦‍♂️. Thanks.